packages/api-elements/lib/define-value-of.js

```diff
@@ -93,10 +93,8 @@ module.exports = (namespace) => {
     return undefined;
   }
 
-  // TODO e instanceof EnumElement fails because of dependency injection
-  // checking for e.element === 'enum' as a temporary  walkaround
-  const isEnumElement = e => (e.element === 'enum' || e instanceof EnumElement);
```

Comment:
The problem being worked around here was solved in a prior PR, you can see the tests continue to pass after this change.